test/parallel/test-crypto-op-during-process-exit.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+const common = require('../common');
+const assert = require('assert');
+const { generateKeyPair } = require('crypto');
+
+// Regression test for a race condition: process.exit() might lead to OpenSSL
+// cleaning up state from the exit() call via calling its destructor, but
+// running OpenSSL operations on another thread might lead to them attempting
+// to initialize OpenSSL, leading to a crash.
+// This test crashes consistently on x64 Linux at the time of writing.
```

Comment:
The test should no longer crash with the accompanying code change?

Reply:
Yes, that’s correct – I wrote the test before the code change… I thought it would be more or less clear that this doesn’t crash anymore because it’s a working test file, but if you any suggestion for how to word this better, I’m happy to take it

Reply:
I'd suggest just removing the last sentence completely but it's not a blocking issue for me.

Reply:
I’ve just changed it to past tense + referred to a specific Node.js version – I do want to mention the OS and platform, because it’s very possible that this doesn’t reproduce on other platforms due to timing differences

Reply:
SGTM thanks.